modules/nw-aws-load-balancer-with-outposts.adoc

```diff
@@ -1,11 +1,26 @@
 // Module included in the following assemblies:
 // * networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc
+// * post_installation_configuration/configuring-aws-outposts.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="nw-aws-load-balancer-with-outposts_{context}"]
-= AWS Load Balancer Operator and AWS Outposts
+= Using the AWS Load Balancer Operator in an AWS VPC cluster extended into an Outpost
 
-AWS Load Balancer Operator can work with AWS Outposts. Application Load Balancer is supported. Network Load balancer is not, so AWS Load Balancer Operator does not provision it on AWS Outposts subnets. Ingress resources must be annotated with the either outposts or regular subnets (but not both).
+You can configure the AWS Load Balancer Operator to provision an AWS Application Load Balancer in an AWS VPC cluster extended into an Outpost.
+AWS Outposts does not support AWS Network Load Balancers.
+As a result, the AWS Load Balancer Operator cannot provision Network Load Balancers in an Outpost.
+
+You can create an AWS Application Load Balancer either in the cloud subnet or in the Outpost subnet.
+An Application Load Balancer in the cloud can attach to cloud-based compute nodes and an Application Load Balancer in the Outpost can attach to edge compute nodes.
+You must annotate Ingress resources with the Outpost subnet or the VPC subnet, but not both.
```

Comment:
This all seems like important info, but I found it hard to follow. Suggestion to take or leave for lightly restructuring this intro:

```suggestion
When you configure the AWS Load Balancer Operator to provision an AWS Application Load Balancer in an AWS VPC cluster extended into an Outpost, you must ensure that you annotate the `Ingress` resources with either the Outpost subnet or the VPC subnet, but not both.

Consider the following points when selecting a subnet:

* AWS Network Load Balancers are not supported by AWS Outposts. As a result, the AWS Load Balancer Operator cannot provision Network Load Balancers in an Outpost.
* You can create an AWS Application Load Balancer in the cloud subnet, where it can attach to cloud-based compute nodes, or in the Outpost subnet, where it can attach to edge compute nodes.
```

Also, not part of your PR, but the step and the config block title both need backticks around `Ingress`